server/src/engineModel/EngineModel.ts

```diff
@@ -92,6 +93,15 @@ export class EngineModel {
                 this.completionProvider.addCompletionItem(method);
             });
         }
+
+        if (!!yclass.instances && !!yclass.instances[0].item) {
+            yclass.instances[0].item.forEach((_instance) => {
```

Comment:
So we’ve got an array of instances, and each one contains an “item” field containing… several instances? It sounds oddly bidimensional, but that’s probably me missing something again.